diff --git a/node_modules/vuepress-theme-reco/components/Footer.vue b/node_modules/vuepress-theme-reco/components/Footer.vue
index 9512a3b..3ab2619 100644
--- a/node_modules/vuepress-theme-reco/components/Footer.vue
+++ b/node_modules/vuepress-theme-reco/components/Footer.vue
@@ -2,7 +2,9 @@
   <div class="footer-wrapper">
     <span>
       <reco-icon icon="reco-theme" />
-      <a target="blank" href="https://vuepress-theme-reco.recoluan.com">{{`vuepress-theme-reco@${version}`}}</a>
+      <a target="blank" href="https://vuepress-theme-reco.recoluan.com">{{
+        `vuepress-theme-reco@${version}`
+      }}</a>
     </span>
     <span v-if="$themeConfig.record">
       <reco-icon icon="reco-beian" />
@@ -13,7 +15,13 @@
       <a>
         <span v-if="$themeConfig.author">{{ $themeConfig.author }}</span>
         &nbsp;&nbsp;
-        <span v-if="$themeConfig.startYear && $themeConfig.startYear != (new Date().getFullYear())">{{ $themeConfig.startYear }} - </span>
+        <span
+          v-if="
+            $themeConfig.startYear &&
+            $themeConfig.startYear != new Date().getFullYear()
+          "
+          >{{ $themeConfig.startYear }} -
+        </span>
         {{ new Date().getFullYear() }}
       </a>
     </span>
@@ -22,73 +30,84 @@
       <AccessNumber idVal="/" />
     </span>
     <p class="cyber-security" v-if="$themeConfig.cyberSecurityRecord">
-      <img src="https://img.alicdn.com/tfs/TB1..50QpXXXXX7XpXXXXXXXXXX-40-40.png" alt="">
-      <a :href="$themeConfig.cyberSecurityLink || '#'">{{ $themeConfig.cyberSecurityRecord }}</a>
+      <img
+        src="https://img.alicdn.com/tfs/TB1..50QpXXXXX7XpXXXXXXXXXX-40-40.png"
+        alt=""
+      />
+      <a :href="$themeConfig.cyberSecurityLink || '#'">{{
+        $themeConfig.cyberSecurityRecord
+      }}</a>
     </p>
-    <Comments :isShowComments="false"/>
+    <Comments :isShowComments="false" />
   </div>
 </template>
 
 <script>
-import { defineComponent, computed } from 'vue-demi'
-import { RecoIcon } from '@vuepress-reco/core/lib/components'
-import { version } from '../package.json'
-import { useInstance } from '@theme/helpers/composable'
+import { defineComponent, computed } from "vue-demi";
+import { RecoIcon } from "@vuepress-reco/core/lib/components";
+import { version } from "../package.json";
+import { useInstance } from "@theme/helpers/composable";
 
 export default defineComponent({
   components: { RecoIcon },
-  setup (props, ctx) {
-    const instance = useInstance()
+  setup(props, ctx) {
+    const instance = useInstance();
     const showAccessNumber = computed(() => {
       const {
         $themeConfig: { valineConfig },
-        $themeLocaleConfig: { valineConfig: valineLocalConfig }
-      } = instance
+        $themeLocaleConfig: { valineConfig: valineLocalConfig },
+      } = instance;
 
-      const vc = valineLocalConfig || valineConfig
+      const vc = valineLocalConfig || valineConfig;
 
-      return vc && vc.visitor != false
-    })
-    return { version, showAccessNumber }
-  }
-})
+      return vc && vc.visitor != false;
+    });
+    return { version, showAccessNumber };
+  },
+});
 </script>
 
 <style lang="stylus" scoped>
-  .footer-wrapper {
-    padding: 1.5rem 2.5rem;
-    border-top: 1px solid var(--border-color);
-    text-align: center;
-    color: lighten($textColor, 25%);
-    a {
-      font-size 14px
+.footer-wrapper {
+  padding: 1.5rem 2.5rem;
+  border-top: 1px solid var(--border-color);
+  text-align: center;
+  color: lighten($textColor, 25%);
+
+  a {
+    font-size: 14px;
+  }
+
+  > span {
+    margin-left: 1rem;
+
+    > i {
+      margin-right: 0.5rem;
     }
-    > span {
-      margin-left 1rem
-      > i {
-        margin-right .5rem
-      }
+  }
+
+  .cyber-security {
+    img {
+      margin-right: 0.5rem;
+      width: 20px;
+      height: 20px;
+      vertical-align: middle;
     }
-    .cyber-security {
-      img {
-        margin-right .5rem
-        width 20px
-        height 20px
-        vertical-align middle
-      }
-      a {
-        vertical-align middle
-      }
+
+    a {
+      vertical-align: middle;
     }
   }
+}
 
 @media (max-width: $MQMobile) {
   .footer {
-    text-align: left!important;
+    text-align: left !important;
+
     > span {
-      display block
-      margin-left 0
-      line-height 2rem
+      display: block;
+      margin-left: 0;
+      line-height: 2rem;
     }
   }
 }
diff --git a/node_modules/vuepress-theme-reco/components/NoteAbstractItem.vue b/node_modules/vuepress-theme-reco/components/NoteAbstractItem.vue
index 1fc00ec..b6f9c79 100644
--- a/node_modules/vuepress-theme-reco/components/NoteAbstractItem.vue
+++ b/node_modules/vuepress-theme-reco/components/NoteAbstractItem.vue
@@ -2,91 +2,166 @@
   <div
     class="abstract-item"
     @click="$router.push(item.path)">
-    <reco-icon v-if="item.frontmatter.sticky" icon="reco-sticky" />
-    <div class="title">
-      <reco-icon v-if="item.frontmatter.keys" icon="reco-lock" />
-      <router-link :to="item.path">{{item.title}}</router-link>
+    <i v-if="item.frontmatter.sticky" class="iconfont reco-sticky"></i>
+    <div class="cover">
+      <img class="cover-img" :src="item.frontmatter.cover
+      || 'https://pan.zealsay.com/zealsay/cover/1.jpg'" :alt="item.title"/>
+    </div>
+    <div class="info">
+      <div class="title">
+        <i v-if="item.frontmatter.keys" class="iconfont reco-lock"></i>
+        <router-link :to="item.path">{{item.title}}</router-link>
+      </div>
+      <div class="abstract" v-html="item.excerpt"></div>
+      <PageInfo
+        :pageInfo="item"
+        :currentTag="currentTag">
+      </PageInfo>
     </div>
-    <div class="abstract" v-html="item.excerpt"></div>
-    <PageInfo
-      :pageInfo="item"
-      :currentTag="currentTag">
-    </PageInfo>
   </div>
 </template>
 
 <script>
-import { defineComponent } from 'vue-demi'
-import { RecoIcon } from '@vuepress-reco/core/lib/components'
-import PageInfo from './PageInfo'
-export default defineComponent({
-  components: { PageInfo, RecoIcon },
-  props: ['item', 'currentPage', 'currentTag']
-})
+  import PageInfo from './PageInfo'
+
+  export default {
+    components: {PageInfo},
+    props: ['item', 'currentPage', 'currentTag']
+  }
 </script>
 
 <style lang="stylus" scoped>
-.abstract-item
-  position relative
-  margin: 0 auto 20px;
-  padding: 16px 20px;
-  width 100%
-  overflow: hidden;
-  border-radius: $borderRadius
-  box-shadow: var(--box-shadow);
-  box-sizing: border-box;
-  transition all .3s
-  background-color var(--background-color)
-  cursor: pointer;
-  > * {
-    pointer-events: auto;
-  }
-  .reco-sticky
-    position absolute
-    top 0
-    left 0
-    display inline-block
-    color $accentColor
-    font-size 2.4rem
-  &:hover
-    box-shadow: var(--box-shadow-hover)
-  .title
-    position: relative;
-    font-size: 1.28rem;
-    line-height: 46px;
-    display: inline-block;
-    a
-      color: var(--text-color);
-    .reco-lock
-      font-size 1.28rem
-      color $accentColor
-    &:after
-      content: "";
-      position: absolute;
-      width: 100%;
-      height: 2px;
-      bottom: 0;
-      left: 0;
-      background-color: $accentColor;
-      visibility: hidden;
-      -webkit-transform: scaleX(0);
-      transform: scaleX(0);
-      transition: .3s ease-in-out;
-    &:hover a
+  .abstract-item
+    position relative
+    display inline-flex
+    margin: 0 auto 20px;
+    padding: 16px 20px;
+    width 100%
+    overflow: hidden;
+    border-radius: $borderRadius
+    box-shadow: var(--box-shadow);
+    box-sizing: border-box;
+    transition all .3s
+    background-color var(--background-color)
+    cursor: pointer;
+    .abstract
+      margin 0 1rem
+
+    > * {
+      pointer-events: auto;
+    }
+
+    .cover
+      max-width 320px
+      flex 1
+      overflow hidden
+      border-radius .5rem
+    .cover-img
+      border-radius .5rem
+      max-width 320px
+      transition: .5s ease-out;
+
+    .cover-img:hover
+      transform: scale3d(1.3, 1.3, 1.3);
+
+    .info
+      flex 1
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+
+    .reco-sticky
+      position absolute
+      top 0
+      left 0
+      display inline-block
       color $accentColor
-    &:hover:after
-      visibility visible
-      -webkit-transform: scaleX(1);
-      transform: scaleX(1);
-  .tags
-    .tag-item
-      &.active
+      font-size 2.4rem
+
+    &:hover
+      box-shadow: var(--box-shadow-hover)
+
+    .title
+      position: relative;
+      font-size: 1.28rem;
+      line-height: 46px;
+      display: inline-block;
+      margin 0 2rem
+
+      a
+        color: var(--text-color);
+
+      .reco-lock
+        font-size 1.28rem
         color $accentColor
-      &:hover
+
+      &:after
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 2px;
+        bottom: 0;
+        left: 0;
+        background-color: $accentColor;
+        visibility: hidden;
+        -webkit-transform: scaleX(0);
+        transform: scaleX(0);
+        transition: .3s ease-in-out;
+
+      &:hover a
         color $accentColor
-@media (max-width: $MQMobile)
-  .tags
-    display block
-    margin-top 1rem;
-    margin-left: 0!important;
+
+      &:hover:after
+        visibility visible
+        -webkit-transform: scaleX(1);
+        transform: scaleX(1);
+
+    .tags
+      .tag-item
+        &.active
+          color $accentColor
+
+        &:hover
+          color $accentColor
+
+  @media (max-width: $homePageWidth)
+    .cover
+      max-width 200px
+      flex 1
+      overflow hidden
+      border-radius .5rem
+      display flex
+      align-items: center
+      .cover-img
+        border-radius .5rem
+        max-width 200px
+        transition: 1s ease-out;
+    .info
+      flex 2 !important
+  @media (max-width: 1080px)
+    .cover
+      display none
+  @media (max-width: $MQMobile)
+    .tags
+      display block
+      margin-top 1rem;
+      margin-left: 0 !important;
+
+    .abstract-item
+      display block
+      text-align center
+
+      .cover
+        width 100%
+        display inline-flex
+        max-width 320px
+        .cover-img
+          max-width 320px
+          width inherit
+
+      .info
+        .title
+          margin 0
+
+
 </style>
diff --git a/node_modules/vuepress-theme-reco/components/PageInfo.vue b/node_modules/vuepress-theme-reco/components/PageInfo.vue
index d28c32b..461cede 100644
--- a/node_modules/vuepress-theme-reco/components/PageInfo.vue
+++ b/node_modules/vuepress-theme-reco/components/PageInfo.vue
@@ -1,105 +1,128 @@
 <template>
-  <div>
-    <reco-icon
-      v-if="pageInfo.frontmatter.author || $themeConfig.author"
-      icon="reco-account"
-    >
-      <span>{{ pageInfo.frontmatter.author || $themeConfig.author }}</span>
-    </reco-icon>
-    <reco-icon
+  <div class="page-info">
+    <i
+      class="iconfont reco-account"
+      v-if="pageInfo.frontmatter.author || $themeConfig.author || $site.title">
+      <span>{{ pageInfo.frontmatter.author || $themeConfig.author || $site.title }}</span>
+    </i>
+    <i
       v-if="pageInfo.frontmatter.date"
-      icon="reco-date"
-    >
-      <span>{{ formatDateValue(pageInfo.frontmatter.date) }}</span>
-    </reco-icon>
-    <reco-icon
+      class="iconfont reco-date">
+      <span>{{ pageInfo.frontmatter.date | formatDateValue }}</span>
+    </i>
+    <i
       v-if="showAccessNumber === true"
-      icon="reco-eye"
-    >
-      <AccessNumber :idVal="pageInfo.path" :numStyle="numStyle" />
-    </reco-icon>
-    <reco-icon
+      class="iconfont reco-eye">
+      <AccessNumber
+        :idVal="pageInfo.path"
+        :numStyle="numStyle"/>
+    </i>
+    <i
       v-if="pageInfo.frontmatter.tags"
-      icon="reco-tag"
-      class="tags"
-    >
+      class="iconfont reco-tag tags">
       <span
         v-for="(subItem, subIndex) in pageInfo.frontmatter.tags"
         :key="subIndex"
         class="tag-item"
         :class="{ 'active': currentTag == subItem }"
-        @click.stop="goTags(subItem)"
-      >{{subItem}}</span>
-    </reco-icon>
+        @click.stop="goTags(subItem)">{{subItem}}</span>
+    </i>
   </div>
 </template>
 
 <script>
-import { defineComponent } from 'vue-demi'
-import { RecoIcon } from '@vuepress-reco/core/lib/components'
-import { useInstance } from '@theme/helpers/composable'
+  // 引入时间格式化js文件
+  import {formatDate} from '@theme/helpers/utils'
 
-export default defineComponent({
-  components: { RecoIcon },
-  props: {
-    pageInfo: {
-      type: Object,
-      default () {
-        return {}
+  export default {
+    props: {
+      pageInfo: {
+        type: Object,
+        default() {
+          return {}
+        }
+      },
+      currentTag: {
+        type: String,
+        default: ''
+      },
+      showAccessNumber: {
+        type: Boolean,
+        default: false
       }
     },
-    currentTag: {
-      type: String,
-      default: ''
+    data() {
+      return {
+        numStyle: {
+          fontSize: '.9rem',
+          fontWeight: 'normal',
+          color: '#999'
+        }
+      }
     },
-    showAccessNumber: {
-      type: Boolean,
-      default: false
-    }
-  },
-
-  setup (props, ctx) {
-    const instance = useInstance()
-
-    const numStyle = {
-      fontSize: '.9rem',
-      fontWeight: 'normal',
-      color: '#999'
-    }
-
-    const goTags = (tag) => {
-      if (instance.$route.path !== `/tag/${tag}/`) {
-        instance.$router.push({ path: `/tag/${tag}/` })
+    filters: {
+      formatDateValue(value) {
+        if (!value) return ''
+        // 返回的value的值都是这个样子2019-09-20T18:22:30.000Z
+        // 对value进行处理
+        value = value.replace('T', ' ').slice(0, value.lastIndexOf('.'))
+        // 转化后的value 2019-09-20 18:22:30
+        // 获取到时分秒
+        const h = Number(value.substr(11, 2))
+        const m = Number(value.substr(14, 2))
+        const s = Number(value.substr(17, 2))
+        // 判断时分秒是不是 00:00:00 (如果是用户手动输入的00:00:00也会不显示)
+        if (h > 0 || m > 0 || s > 0) {
+          // 时分秒有一个> 0 就说明用户输入一个非 00:00:00 的时分秒
+          return formatDate(value)
+        } else {
+          // 用户没有输入或者输入了 00:00:00
+          return formatDate(value, 'yyyy-MM-dd')
+        }
+      }
+    },
+    methods: {
+      goTags(tag) {
+        if (this.$route.path !== `/tag/${tag}/`) {
+          this.$router.push({path: `/tag/${tag}/`})
+        }
       }
     }
-
-    const formatDateValue = (value) => {
-      return new Intl.DateTimeFormat(instance.$lang).format(new Date(value))
-    }
-
-    return { numStyle, goTags, formatDateValue }
   }
-})
 </script>
 
 <style lang="stylus" scoped>
-.iconfont
-  display inline-block
-  line-height 1.5rem
-  &:not(:last-child)
-    margin-right 1rem
-  span
-    margin-left 0.5rem
-.tags
-  .tag-item
-    font-family Ubuntu, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif
-    cursor pointer
-    &.active
-      color $accentColor
-    &:hover
-      color $accentColor
-@media (max-width: $MQMobile)
+  .page-info
+    margin 0 2rem
+    display inline-flex
+    flex-wrap wrap
+  .iconfont
+    display inline-block
+    line-height 1.5rem
+
+    &:not(:last-child)
+      margin-right 1rem
+
+    span
+      margin-left 0.5rem
+
   .tags
-    display block
-    margin-left 0 !important
+    .tag-item
+      font-family Ubuntu, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif
+      cursor pointer
+
+      &.active
+        color $accentColor
+
+      &:hover
+        color $accentColor
+
+  @media (max-width: $MQMobile)
+    .tags
+      display block
+      margin-left 0 !important
+    .page-info
+      margin 0
+      justify-content center
 </style>
+
